.github/workflows/lambda.yml

```diff
@@ -16,15 +16,24 @@ jobs:
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Install docker
+        run: |
+          brew install docker docker-machine docker-compose
+          sudo docker --version
+      - run: colima start # run docker
+      - run: docker-compose up -d testagent # Start the testagent container in the background
```

Comment:
This should use the built-in functionality of GHA instead of Compose.

Reply:
Do you mean to use the 'services' keyword similar to some of the plugin tests?

Reply:
Yes, using Compose should almost never be needed in GitHub Actions except in very rare edge cases.

Reply:
Understood. I changed one of the plugin tests so far (amqp10) to use the docker services available within Github Actions. My only concern is the logging isn't quite as well formatted as using compose. [This](https://github.com/DataDog/dd-trace-js/pull/2887/files#diff-660fbacbcd30e79ad55c4c69af9a034da3af7f8a37f8704d72b79617a93ee864:~:text=%2D%205673%3A5672-,testagent%3A,testagent.id%20%7D%7D%20%23%20Display%20the%20log%20output%20of%20the%20testagent%20container,-%2D%20uses%3A) is the change I made to use the services keyword. See [here for resulting logs](https://github.com/DataDog/dd-trace-js/actions/runs/4883247660/jobs/8714330518?pr=2887#step:12:11) with that approach. See [here for logs using docker-compose](https://github.com/DataDog/dd-trace-js/actions/runs/4883247660/jobs/8714329254?pr=2887#step:13:12), which IMO are much easier to read.